redfish/provider/resource_redfish_dell_system_attributes_test.go

```diff
@@ -0,0 +1,155 @@
+package provider
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccRedfishSystemAttributesBasic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccRedfishResourceSystemAttributesConfig(creds),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("redfish_dell_system_attributes.system", "attributes.SupportInfo.1.Outsourced", "Yes"),
+					resource.TestCheckResourceAttr("redfish_dell_system_attributes.system", "attributes.ServerPwr.1.PSPFCEnabled", "Disabled"),
+				),
+			},
+			{
+				Config: `resource "redfish_dell_system_attributes" "system" {
+				}`,
+				ResourceName:  "redfish_dell_system_attributes.system",
+				ImportState:   true,
+				ImportStateId: "{\"attributes\":[\"ServerPwr.1.PSPFCEnabled\"],\"username\":\"" + creds.Username + "\",\"password\":\"" + creds.Password + "\",\"endpoint\":\"https://" + creds.Endpoint + "\",\"ssl_insecure\":true}",
```

Comment:
This test seems contradictory by itself since the actual resource and imported resource attributes are supposed to be different.
Please set `ImportStateVerify` to true, remove the `Config` field and check if the test passes.

Reply:
changed the test case